.gitattributes

```diff
@@ -16,3 +16,5 @@
 SciLexer.dll filter=lfs diff=lfs merge=lfs -text
 *.psd filter=lfs diff=lfs merge=lfs -text
 *.FH10 filter=lfs diff=lfs merge=lfs -text
+*.sql filter=lfs diff=lfs merge=lfs -text
```

Comment:
Only the .mdb entry should be in this file. 
I forgot to add this .gitattributes file to the commit when I put the mdb into the repo as an LFS file, so that may be why you've ended up committing it.